features/fetch-request-streaming.yml

```
@@ -0,0 +1,5 @@
+name: Fetch request streaming
```

I don't love gerund feature names but avoiding ambiguity here is hard. After reading a bunch of StackOverflow questions and answers, I think this might be the least-worst option. What do you think of this alternative?

```suggestion
name: Fetch request streams
```

(This would require updating the ID too.)